charts/app-config/templates/draft-content-store-proxy-service.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.enableDraftContentStoreProxy }}
+# route all requests for draft-content-store via draft-content-store-proxy
+apiVersion: v1
+kind: Service
+metadata:
+  name: draft-content-store
+spec:
+  ports:
+    - name: app
+      port: 80
+      targetPort: 8080
+  selector:
+    app: draft-content-store-proxy
```

Comment:
You probably want a service of type [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname), rather than duplicating all the endpoints and so on. It'd also be clearer that this is an alias that way.

(Does naming the Argo app for the proxy `content-store` not quite do what you want?)

Reply:
I've tried with an `ExternalName` service type (it does look simpler that way)

Reply:
> (Does naming the Argo app for the proxy content-store not quite do what you want?)

I thought that might be more confusing in Argo and so on, so I wanted to try this way first. If this doesn't work, that's the next thing to try.

Reply:
Ah I see, fair enough :)